cmake/FindLibkvm.cmake

```diff
@@ -0,0 +1,15 @@
+# Try to find Libkvm functionality
+# Once done this will define
+#
+#  LIBKVM_FOUND - system has Libkvm
+#  LIBKVM_INCLUDE_DIR - Libkvm include directory
+#  LIBKVM_LIBRARIES - Libraries needed to use Libkvm
+#
+
+find_path(LIBKVM_INCLUDE_DIR kvm.h)
+find_library(LIBKVM_LIBRARIES NAMES kvm)
+
+include(FindPackageHandleStandardArgs)
```

Comment:
Can you make this using modern cmake feature?
See https://github.com/fcitx/fcitx5/blob/5c0e52b4e8410de3c300feeff28ec78c7be056ac/cmake/FindDL.cmake#L40 for an example

Reply:
Done.